appveyor.yml

```diff
@@ -7,7 +7,9 @@ os: Visual Studio 2015
 environment:
   matrix:
     - CMAKE_VS_VERSION: "10 2010"
+      TEST_ALL: "no"
```

Comment:
This makes appveyor stop running as soon as it encounters an error. Useful as I'm developing to make appveyor churn through failed tests faster. I can remove this before merging if you want.